frontend/src/metabase/dashboard/selectors.spec.js

```diff
@@ -0,0 +1,132 @@
+import { getParameters } from "./selectors";
+
+import { chain } from "icepick";
+
+const STATE = {
+    dashboard: {
+        dashboardId: 0,
+        dashboards: {
+            0: {
+                ordered_cards: [0, 1],
+                parameters: []
+            }
+        },
+        dashcards: {
+            0: {
+                card: { id: 0 },
+                parameter_mappings: []
+            },
+            1: {
+                card: { id: 1 },
+                parameter_mappings: []
+            }
+        }
+    },
+    metadata: {
+        databases: {},
+        tables: {},
+        fields: {},
+    }
+}
+
+fdescribe("dashboard/selectors", () => {
```

Comment:
Oh, yeah good catch, I guess we don't have "jasmine/no-focused-tests" lint rule enabled for jest tests.